yt/visualization/fixed_resolution.py

```diff
@@ -708,21 +712,37 @@ def __getitem__(self, item):
             deposition,
         )
 
-        bounds = []
-        for b in self.bounds:
-            if hasattr(b, "in_units"):
-                b = float(b.in_units("code_length"))
-            bounds.append(b)
+        dd = self.data_source.dd
 
         ftype = item[0]
-        x_data = self.data_source.dd[ftype, self.x_field]
-        y_data = self.data_source.dd[ftype, self.y_field]
-        data = self.data_source.dd[item]
+        if self.axis is None:
+            wd = tuple(el.in_units("code_length").v for el in self.data_source.width)
+            x_data, y_data, rbx0, rbx1, rby0, rby1 = rotate_particle_coord(
+                dd[ftype, "particle_position_x"].to_value("code_length"),
+                dd[ftype, "particle_position_y"].to_value("code_length"),
+                dd[ftype, "particle_position_z"].to_value("code_length"),
+                self.data_source.center.to("code_length").d,
+                wd,
+                self.data_source.normal_vector,
+                self.data_source.north_vector,
+            )
+            bounds = [rbx0, rbx1, rby0, rby1]
+            x_data = np.array(x_data)
+            y_data = np.array(y_data)
+        else:
+            bounds = []
+            for b in self.bounds:
+                if hasattr(b, "in_units"):
+                    b = float(b.in_units("code_length"))
+                bounds.append(b)
+            x_data = dd[ftype, self.x_field].to_value("code_length")
+            y_data = dd[ftype, self.y_field].to_value("code_length")
+        data = dd[item]
 
         # handle periodicity
-        dx = x_data.in_units("code_length").d - bounds[0]
-        dy = y_data.in_units("code_length").d - bounds[2]
-        if self.periodic:
+        dx = x_data - bounds[0]
+        dy = y_data - bounds[2]
+        if self.axis is not None and self.periodic:
```

Comment:
`self.periodic` is a tuple, right?  I know this code predates the PR but shouldn't this be an `all` or `self.periodic[self.axis]` (well probably not that) or something?

Reply:
Actually here it's just a scalar, I checked (you have to kind of dive down but it's in the signature for the `PlotWindow` class.